### PR TITLE
removed control directory - no longer needed

### PR DESCRIPTION
**Pull request by mvertens**
Summary: removal of control directory - it is no longer needed. 

In addition the following was also done:
- removed call to cam_export_uv and use of cam_export_uv since it was not being used for anything.
- changed then name of AOD_VIS to AODVIS to permit easier comparison between oslo-aero and tropmam4

Contributors: mvertens

Reviewers: @gold2718 

Purpose of changes: Minimize number of files in camnor_physics and remove duplicate files where possible

Changes made to build system: None

Changes made to the namelist: None

Changes to the defaults for the boundary datasets: None

Substantial timing or memory changes: None

Testing:
With the code for the above two PRs - ran aux_cam_noresm on betzy, compared to noresm_v10_cam6_3_123 and generated noresm_v11_cam6_3_123
All tests passed
Baselines are different compared to noresm_v10_cam6_3_123 for the following:
ERP_D_Ln9_P18.ne3pg3_ne3pg3_mg37.NFLTHISToslo_v0d.betzy_intel.cam-outfrq9s
ERP_Ln9.ne30pg3_ne30pg3_mtn14.NF1850oslo.betzy_intel.cam-outfrq9s
ERP_Ln9.ne30pg3_ne30pg3_mtn14.NF1850tropmam4.betzy_intel.cam-outfrq9s
ERP_Ln9.ne30pg3_ne30pg3_mtn14.NF2000oslo.betzy_intel.cam-outfrq9s
ERP_Ln9.ne30pg3_ne30pg3_mtn14.NF2000tropmam4.betzy_intel.cam-outfrq9s
SMS_D_Ln9_P18.ne3pg3_ne3pg3_mg37.NF2000oslo.betzy_intel.cam-outfrq9s

### CODE REVIEW

**Comment by mvertens** [comment]
@gold2718 - could you please review this - I need it to be consistent with the latest changes I am making to the use_cases and rewrite of the oslo_aero prescribed_volcaero.

File: physics/physpkg.F90 [hunk 1]
```diff
@@ -2652,6 +2652,7 @@ subroutine tphysbc (ztodt, state,  &
     real(r8) :: zero_tracers(pcols,pcnst)
 
     logical   :: lq(pcnst)
+    logical   :: first_call = .true.
```
Comment (gold2718) [inline physics/physpkg.F90]: This is not used, please remove.

**Review by gold2718 — CHANGES_REQUESTED** [review]
One simple change needed.

File: physics/physpkg.F90 (hunk 1 above)
Comment (mvertens) [reply physics/physpkg.F90]: Done.

**Comment by mvertens** [comment]
@gold2718 - can you please merge this - or should I?

**Comment by gold2718** [comment]
> gold2718 - can you please merge this - or should I?

Merge is done by the assignee, _after_ it is approved.

**Review by gold2718 — APPROVED** [review]
Looks good now.

**Pull request merged**